RecoEgamma/EgammaElectronAlgos/src/GsfElectronAlgo.cc

```diff
@@ -315,7 +316,13 @@ reco::GsfElectron::ShowerShape GsfElectronAlgo::calculateShowerShape(const reco:
   }
 
   std::vector<float> covariances = ClusterTools::covariances(seedCluster, recHits, &topology, &geometry);
-  std::vector<float> localCovariances = ClusterTools::localCovariances(seedCluster, recHits, &topology);
+
+  // do noise-cleaning for full5x5, by passing per crystal PF recHit thresholds and mult values
+  // mult values for EB and EE were obtained by dedicated studies
+  std::vector<float> localCovariances =
+      full5x5 ? ClusterTools::localCovariances(seedCluster, recHits, &topology, 4.7, &thresholds, 1.0, 1.25)
```

Comment:
These constants 4.7, 1.0, 1.25 seem to be repeated in a few places throughout this changeset. Is there a common location you can import the values from?

Reply:
I created this new file, 
RecoEgamma/EgammaTools/interface/EgammaLocalCovParamDefaults.h 
where I have these constants now, and importing their values to other codes from here